src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml

```diff
@@ -176,22 +174,14 @@ tests:
               minValue: 5
```

Comment:
hasMasksSet:  should be used here instead of ranges

Reply:
Updated